src/org/labkey/test/components/ui/grids/EditableGrid.java

```diff
@@ -754,20 +754,22 @@ private void selectCell(WebElement cell)
     private void activateCell(WebElement cell)
     {
         // If it is a selector, and it already has focus (is active), it will not have a div.cellular-display
-        if(Locator.tagWithClass("div", "select-input__control--is-focused")
-                .findElements(cell).isEmpty())
-        {
-            var cellContent = Locator.tagWithClass("div", "cellular-display").findElement(cell);
-            cellContent.sendKeys(Keys.ENTER);
-        }
+        if (Locator.tagWithClass("div", "select-input__control--is-focused").findElements(cell).isEmpty())
+            sendKeysToCell(cell, Keys.ENTER);
+    }
+
+    public void sendKeysToCell(WebElement cell, CharSequence... keysToSend)
```

Comment:
May want to add a comment here as to the value of this method. For example does it emulate the barcode scanner?